### PR TITLE
Lowers threat level of magical hats from 6 to 4 for securitron threat assessment

### PR DESCRIPTION
**Pull request by silicons**
-_-

### CODE REVIEW

**Comment by lolman360** [comment]
based pr

**Review by nik707 — APPROVED** [review]

**Comment by nik707** [comment]
haha get fucked idiot fix this

**Review by nik707 — CHANGES_REQUESTED** [review]
fix conflict

**Comment by Poojawa** [comment]
i ded

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1,1046 +1,1046 @@
...
+											to_chat(usr, "<b>Crime:</b> [c.crimeName]")
+											to_chat(usr, "<b>Details:</b> [c.crimeDetails]")
+											to_chat(usr, "Added by [c.author] at [c.time]")
+											to_chat(usr, "----------")
+										to_chat(usr, "<b>Major Crimes:</b>")
+										for(var/datum/data/crime/c in R.fields["ma_crim"])
+											to_chat(usr, "<b>Crime:</b> [c.crimeName]")
+											to_chat(usr, "<b>Details:</b> [c.crimeDetails]")
+											to_chat(usr, "Added by [c.author] at [c.time]")
+
```
Comment (Poojawa) [inline code/modules/mob/living/carbon/human/human.dm:602]: You should really just make your anime hat a different item, then you wouldn't need to buff syndicates and wizards

**Review by deathride58 — CHANGES_REQUESTED** [review]
fix the line endings plz

**Pull request closed without merging**